public/app/features/teams/state/actions.ts

```diff
@@ -43,6 +44,20 @@ export function loadTeams(initial = false): ThunkResult<void> {
   };
 }
 
+export function loadUsersTeams(): ThunkResult<void> {
+  return async (dispatch) => {
+    // Early return if the user cannot list teams
+    if (!contextSrv.hasPermission(AccessControlAction.ActionTeamsRead)) {
```

Comment:
nit: if the user does not have access to teams then they would see that they are not part of a team. Or do they actually get an error saying that they do not have access? 🤔